scripts/ci/jobs/ocp_qa_e2e_tests.py

```diff
@@ -15,4 +15,10 @@
 os.environ["ROX_RISK_REPROCESSING_INTERVAL"] = "15s"
 os.environ["ROX_SENSOR_CONNECTION_RETRY_MAX_INTERVAL"] = "30s"
 
+for v in ['JOB_NAME_SAFE', 'CLUSTER_FLAVOR_VARIANT', 'CLUSTER_PROFILE_NAME', 'CLUSTER_TYPE', 'OPENSHIFT_CI_STEP_NAME']:
+    print(v + ':' + os.environ.get(v, ''))
+
+if 'openshift-4' in os.environ.get('CLUSTER_FLAVOR_VARIANT', ''):
```

Comment:
I'm guessing this means `CLUSTER_FLAVOR_VARIANT` is only set for the interop job?